src/amo/routes.js

```diff
@@ -39,6 +39,8 @@ export default (
       <IndexRoute component={Home} />
       <Route path="addon/:slug/" component={Addon} />
       <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
+      {config.get('enableUserProfile') &&
```

Comment:
Is this necessary? I don't think the URL is accessible on the frontend until we allow it from nginx, right?

Reply:
I guess not, but it seemed more complete/proper. I can remove it, but I just figured leaving it here was a bit of code-as-documentation that this is pref'd. I can just leave a comment though, fair enough.